wiki/en/Getting-help.md

```diff
@@ -0,0 +1,16 @@
+If you need help, ask your question in the [Gitter Chat](https://gitter.im/SpaceVim/SpaceVim), a member of the community will help you out.
+
+here is a simple guide for how to getting help about SpaceVim. Before asking question, please read the FAQ, the answer to your question maybe
```

Comment:
English:  
* Either say "for how to get help with SpaceVim" or "for getting help with SpaceVim". 
* Either "before asking a question" or "before asking questions"
* s/maybe/may be/